config/routes.rb

```diff
@@ -611,7 +611,7 @@
           resources :referrer_filters, :controller => 'buyer_application_referrer_filters', :only => [:index, :create, :destroy]
         end
 
-        resources :service_contracts, :only => [:index, :destroy]
+        resources :service_contracts, :only => [:index, :create, :update, :destroy, :show]
```

Comment:
```suggestion
        resources :service_contracts, only: %i[index create update destroy show]
```

Just better Ruby style